src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java

```diff
@@ -5,8 +5,23 @@
 
 package com.microsoft.sqlserver.jdbc;
 
+import java.io.File;

```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.checks.imports.UnusedImportsCheck
**Severity**: ERROR
**File**: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java L8

Unused import - java.io.File.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-1870521805>*